src/loki/reach.cc

```diff
@@ -221,8 +221,11 @@ void Reach::ExpandingNode(baldr::GraphReader&,
 }
 
 // when the main loop is looking to continue expanding we tell it to terminate here
-thor::ExpansionRecommendation
-Reach::ShouldExpand(baldr::GraphReader&, const sif::EdgeLabel&, const thor::ExpansionType) {
+thor::ExpansionRecommendation Reach::ShouldExpand(baldr::GraphReader&,
+                                                  const sif::EdgeLabel&,
+                                                  const thor::ExpansionType,
+                                                  const float /*time*/,
+                                                  const uint32_t /*dist*/) {
```

Comment:
I added those, since the `Isochrone::ShouldExpand` was previously looking at the edge labels, but with multimodal we use `MMEdgeLabel`, not `BDEdgeLabel` so it's either this or templating the `ShouldExpand` function.